plugin/src/index.ts

```diff
@@ -100,6 +97,15 @@ const plugin: NetlifyPlugin = {
       nextConfig: { basePath, i18n, trailingSlash, appDir },
       buildId,
     })
+
+    if (process.env.NEXT_USE_NETLIFY_EDGE) {
+      console.log(outdent`
+        ✨ Deploying to ${greenBright`Netlify Edge Functions`} ✨
+        This feature is in beta. Please share your feedback here: https://ntl.fyi/next-netlify-edge
```

Comment:
This link seems to just point to the Netlify homepage

Reply:
We'll create this when we launch. There's a bullet in the launch checklist